keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/constraints/CanaryConstraintEvaluator.kt

```diff
@@ -57,7 +57,7 @@ import org.springframework.stereotype.Component
 class CanaryConstraintEvaluator(
   private val handlers: List<CanaryConstraintDeployHandler>,
   private val orcaService: OrcaService,
-  override val deliveryConfigRepository: DeliveryConfigRepository,
+  override val combinedRepository: CombinedRepository,
```

Comment:
I'd ❤️ to see this variable just called `repository` throughout every file but nbd.

Reply:
I was wondering if that would be confusing, but since both you and Gal suggested it I'll make the change!